openpype/hosts/maya/api/pipeline.py

```diff
@@ -229,6 +265,12 @@ def _before_scene_save(return_code, client_data):
     )
 
 
+def _remove_workfile_lock():
+    filepath = current_file()
+    if filepath:
+        remove_workfile_lock(filepath)
```

Comment:
```suggestion
def remove_current_workfile_lock():
    """Remove workfile lock on current file"""
    filepath = current_file()
    if filepath:
        remove_workfile_lock(filepath)
```
Does this make more sense?

And feels like this could actually be exposed in the `openpype.pipeline.lock_workfile` too so that potential other hosts who would like to implement this could do so - especially because it just uses the workio API `current_file` function.

Reply:
> especially because it just uses the workio API current_file function

I don't think this is a good idea. Functions in `lock_workfile` are so simple on purpose. Keep in mind that not each host handle workfiles the same way.